tutorial/tutorial.md

```diff
@@ -0,0 +1,398 @@
+```
+
+Restart the control plane
+
+# Run the RHOC Camel operator
+
+In another shell window, navigate to `cos-fleetshard/cos-fleetshard-operator-camel` and set the following environment variables
+
+```
+export COS_OPERATOR_ID=camel-k
+export COS_OPERATOR_VERSION=1.0.0
+export KUBERNETES_NAMESPACE=redhat-openshift-connectors
+```
+
+Now run the operator locally with 
+
+`mvn -Plocal`
+
+The operator will register in the cluster with the name and the namespace provided. Verify it with
+
+```
+kubectl get managedconnectoroperators.cos.bf2.org -A
+NAMESPACE                     NAME      RUNTIME   TYPE                       VERSION
+redhat-openshift-connectors   camel-k   camel-k   camel-connector-operator   1.0.0
+```
+
+## Create the Slack connector
+
+
+Create the needed kafka topic and the kafka service account first. 
+Navigate to a different folder as these commands will create a `./service-acct-credentials.json` file you should avoid putting inside any GitHub repository. 
+You'll need to provide a TOPIC_NAME and a SHORT_DESCRIPTION
+
+```shell
+
+cd ~
+
+rhoas kafka topic create --name=<TOPIC_NAME>
+
+rhoas service-account list
+
+rhoas service-account create --output-file=./service-acct-credentials.json --file-format=json --overwrite --short-description=<SHORT_DESCRIPTION>
+
+rhoas service-account list | grep $USER
+
+export SERVICEACCOUNT=$(cat ./service-acct-credentials.json | jq -r '.clientID')
+
+rhoas kafka acl grant-access --consumer --producer --service-account $SERVICEACCOUNT --topic-prefix <TOPIC_NAME>  --group all
+
+```
+
+In the `cos-tools` tab, save this payload in a file called `slack-example-connector.json`, and edit
+* Kafka id and URL
+* namespace_id (use `cos-tools` `./get-namespaces`)
+* topic name, 
+* service account client_id and client_secret 
+* slack webhook accordingly.
+* slack channel
+
+```json
+{
+  "name": "local-slack-sink",
+  "kind": "ConnectorType",
+  "channels": [
+    "stable"
+  ],
+  "connector_type_id": "slack_sink_0.1",
+  "desired_state": "ready",
+  "kafka": {
+    "id": "KAFKA_ID",
+    "url": "KAFKA_URL"
+  },
+  "namespace_id": "NAMESPACE_ID",
+  "service_account": {
+    "client_id": "CLIENT_ID",
+    "client_secret": "CLIENT_SECRET"
+  },
+  "connector": {
+    "data_shape": {
+      "consumes": {
+        "format": "application/octet-stream"
+      }
+    },
+    "kafka_topic": "KAFKA_TOPIC",
+    "slack_channel": "SLACK_CHANNEL",
+    "slack_webhook_url": "SLACK_WEBHOOK_URL",
+    "error_handler": {
+      "stop": {}
+    }
+  }
+}
+
+```
+
+Create the connector with 
+
+```
+./create-connector "$(cat example-connector.json)"
+```
+
+You should see no logs inside the operator and the `cos-fleetshard-sync`
+And verify it goes to ready state with
+
+```./get-connectors```
+
+
+Now you can access the kafka topic with the following command (it needs `kcat` installed)
```

Comment:
It is still known as `kafkacat` on non-Mac-OS installations: https://github.com/edenhill/kcat#install